### PR TITLE
errorResponse rejected

### PR DESCRIPTION
**Pull request by neeswebservices**

### CODE REVIEW

**Comment by neeswebservices** [comment]
Ok it is ok with it !

File: api/bin/www.js [hunk 1]
```diff
@@ -4,57 +4,62 @@
...
-				server.listen(port)
-				server.on("error", onError)
-				server.on("listening", onListening)
-				console.log("🚀 Listening and MongoDB connected ✅ in port", port)
-			})
-			.catch((err) => {
-				console.log(err)
-				process.exit(0)
-			})
+    ? mongoose
```
Comment (sudipkumarmahato) [inline api/bin/www.js:34]: Ok nice way of handling different mongo database

File: api/bin/www.js [hunk 2]
```diff
@@ -74,53 +79,54 @@ process.env.MONGOURI
...
-			process.exit(1)
-			break
-		case "EADDRINUSE":
-			console.error(bind + " is already in use")
-			process.exit(1)
-			break
-		default:
-			throw error
-	}
+    if (error.syscall !== 'listen') {
```
Comment (sudipkumarmahato) [inline api/bin/www.js:102]: TEst driven

File: api/bin/www.js (hunk 1 above)
Comment (neeswebservices) [reply api/bin/www.js:34]: Thanks

File: api/bin/www.js [hunk 3]
```diff
@@ -4,57 +4,62 @@
...
-const port = normalizePort(process.env.PORT || "8000")
-app.set("port", port)
+const port = normalizePort(process.env.PORT || '8000');
+app.set('port', port);
 
 /**
  * Create HTTP server.
  */
 
-const server = http.createServer(app)
```
Comment (sudipkumarmahato) [inline api/bin/www.js:25]: Ok good server!

**Review by sudipkumarmahato — APPROVED** [review]
Ok now able to merge!

**Pull request merged**